### PR TITLE
Initial website implementation

### PR DESCRIPTION
**Pull request by robinbortlik**
- this commit adds basic setup for Jekyll static site generator using Docker
- initial website design using tailwind, heroicons and daisyui

- still missing some mobile optimisation (looking for help)
- 
Dock-developer-program

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  <div class="flex-col hero-content">
+    <div class="max-w-xl">
+      <img src="https://images.unsplash.com/photo-1520333789090-1afc82db536a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2102&q=80" class="max-w-full rounded-lg shadow-2xl">
+    </div>
+    <div class="max-w-xl">
+      <h1 class="mb-5 text-4xl font-bold">
+        Developer program
+      </h1>
+      <p class="mb-5">
+         has been initiated by Dock community to support growth of Dock ecosystem and reward open source developers for theirs hard work.
```
Comment (mike-parkhill) [inline docs/index.html:17]: We'll probably need to provide the final text to display here.

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+</div>
+
+<div class="mx-auto max-w-7xl w-full text-center mt-8 sm-text-justify">
+  <h2 class="text-3xl tracking-tight font-bold">
+    <span class="block xl:inline">How does it work?</span>
+  </h2>
+  <p class="mt-6 max-w-3xl mx-auto">
+    There are plenty of tasks Dock foundation and community would love to solve. If you are developer then there is nothing simpler then choosing one of the stasks listed below and start work on it.
+  </p>
+  <p class="mt-6 max-w-3xl mx-auto">When you are done, just submit your work to <a class="link" href="mailto:support@dock.io">support@dock.io</a> and receive a reward.</p>
```
[the remark this answers is not in the document]
Comment (robinbortlik) [reply docs/index.html:30]: Yep, that is great idea. :)

**Comment by robinbortlik** [comment]
Hi @mike-parkhill , 

thank you very much for detailed CR. This is basically the proof of concept. I would like to iterate on this PR with you and others until we have minimal version to release. 

I agree with all points you mentioned. All those texts are basically just a placeholders. I plan to update them, based on discuss with you guys :)

**Comment by robinbortlik** [comment]
@mike-parkhill may I ask, did you try to run the website locally? Was it easy to setup? Was everything working on first go?

**Pull request merged**